doc/cmd/scan/supported_resources/aws.md

```diff
@@ -103,6 +103,12 @@ As AWS documentation recommends, the below policy is granting only the permissio
 }
 ```
 
+## CloudFormation template
```

Comment:
Maybe we could put this before the police above ? I think some user may miss this option as policy could be long and some may not scroll to bottom. wdyt ?

Reply:
Indeed, I will put it above the policy.

Reply:
Done 38fb1d3